vim/autoload/vlime/overlay/slimv.vim

```diff
@@ -499,3 +499,6 @@ function! SlimvEval(data)
         call vlime#overlay#slimv#SendToREPL(expr)
     endfor
 endfunction
+
+nnoremap <F2> :<C-U>let g:my_slimv_eval_form=vlime#overlay#slimv#CurTopExprOrAtom() \| call vlime#overlay#slimv#SendToREPL(g:my_slimv_eval_form)<CR>
```

Comment:
These key mappings are not part of Slimv, yet they can be easily implemented in one's `vimrc`, so I'd prefer not to include them in this overlay.